.github/workflows/kentik-api.yml

```diff
@@ -78,8 +78,9 @@ jobs:
         working-directory: ./kentik_api_library
         run: |
           pip3 install -r requirements.txt
+          pip3 install -r requirements-build.txt
```

Comment:
We may as well put the build module in `requirements.txt`. There should never be any additional packages required to run `python -m build` (at least in theory). I cannot imaging a scenario in which a user/developer would benefit from the separate `requirements-build.txt` file.

Reply:
Merging requirements-build.txt into requirements.txt